samples/snapshot/pom.xml

```diff
@@ -41,7 +43,6 @@
     <dependency>
       <groupId>com.google.truth</groupId>
       <artifactId>truth</artifactId>
```

Comment:
Need a version number here.

Reply:
Ah I thought it could also be inherited from parent. Added.